packages/react-widgets/src/models/utils.js

```diff
@@ -0,0 +1,64 @@
+import { AggregationTypes, _filtersToSQL } from '@carto/react-core';
+import { MAP_TYPES, API_VERSIONS } from '@deck.gl/carto';
+
+export function wrapModelCall(props, fromLocal, fromRemote) {
```

Comment:
Nice idea. Maybe I would suggest a simpler name... like `getModel`. 

Also I suggest using ({props, fromLocal, fromRemote}) as params (in general, when more than one param is passed to a method)

Reply:
About the params, whatever you want but I prefer this way haha.
But I'd keep the name, getModel should return fromLocal or fromRemote, and I prefer the fn to make the call. Maybe `callModel` is ok for you?

Reply:
don't worry, it's just something internal, let's keep your current one